docs/resources/kubeletconfig.md

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Please add an example how to use this resource
This example should be added under [generate_example_usages](https://github.com/terraform-redhat/terraform-provider-rhcs/tree/main/generate_example_usages/resources), than the auto generator copy the example here